index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Heather's Portfolio</title>
+</head>
+<body>
+    <h1> Heather Smith</h1>
+    <section id="about">
+    <h2>About</h2>
+    <p>I am currently a flight attendant with a primary background of customer service management. I am a novice to the programming industry, but am looking forward to the avenues this education can open for me. With little prior knowledge, this has been an interesting experience, but one I have embraced in order to persevere!  </p>
+</section>
+    <section id="experience">
+    <h2>Experience</h2>
+    <ul>
+        <li>Pre-work assignment with CTD</li>
+        <li>Current student CTD-Jupiter</li>  
+    </section>
+    <section id="connect">        
+    <h2>Connect</h2>
+    <ul>        
+        <li>https://github.com/Heather813903/Heather813903</li>
```

Comment:
The guidance notes that these need to be link tags. then, if you'd like, they can be wrapped in list item tags, something like this
```html
<li>
  <a href="link-to-site">label for the site</a>
</li>
```